apps/src/lib/node/ledger/shell/finalize_block.rs

```diff
@@ -292,10 +292,9 @@ where
             let (consensus_key, power) = match update {
                 ValidatorSetUpdate::Active(ActiveValidator {
                     consensus_key,
-                    voting_power,
+                    bonded_stake,
                 }) => {
-                    let power: u64 = voting_power.into();
-                    let power: i64 = power
+                    let power: i64 = bonded_stake
```

Comment:
I think this is the only place where we should apply `tm_votes_per_token`

Reply:
why here, since a validator set update is dependent on the bonded stake and not voting power? I realize I should probably change the comments and var names here if we do want to keep it as just `bonded_stake`.

Reply:
This is the validator set that's passed to tendermint (added to the `response.validator_updates` below)

Reply:
Ah I see. Ok, there happens to be one other location where `into_tm_votes_per_token` is currently used, and that is to set a `ferveo_common::TendermintValidator` inside `namada/apps/src/lib/node/ledger/shell/queries.rs`. Should it remain there?